app/uk/gov/hmrc/leakdetection/model/Leak.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.leakdetection.model
+
+import uk.gov.hmrc.leakdetection.scanner.Match
+
+import java.time.Instant
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Reads._
+import play.api.libs.json._
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
+
+case class Leak(
+                     repoName   : String,
+                     branch     : String,
+                     timestamp  : Instant,
+                     reportId   : ReportId,
+                     ruleId     : String,
+                     description: String,
+                     filePath   : String,
+                     scope      : String,
+                     lineNumber : Int,
+                     urlToSource: String,
+                     lineText   : String,
+                     matches    : List[Match],
+                     priority   : Option[String],
```

Comment:
This is only an Option because we've recently introduced it and there were unresolved leaks from before it existed.  Going forward, we should expect all rules to have a priority.  it's fine to leave it as is but if we wanted too, we could make it a String and calculate/default the value during the migration.